docs_src/general/ServicePorts.md

```diff
@@ -4,46 +4,48 @@ The following tables (organized by type of service) capture the default service
 === "Core"
     |Services Name|	Port Definition|
     |---|---|
-    |core-data|	48080|
+    |core-data|	59880|
     | 	|5563|
-    |core-metadata	|48001|
-    |core-command	|48082|
+    |core-metadata	|59881|
+    |core-command	|59882|
 === "Supporting"
     |Services Name|	Port Definition|
     |---|---|
-    |support-notifications	|48060|
-    |support-logging	|48061|
-    |support-scheduler|	48085|
+    |support-notifications	|59860|
+    |support-scheduler|	59861|
 === "Application & Analytics"
     |Services Name|	Port Definition|
     |---|---|
-    |app-service-rules|48095|
-    |rules engine/Kuiper|48075|
-    |   |20498|
+    |app-service-rules|59701|
+    |rules engine/Kuiper|59720|
+    |app-rfid-llrp-inventory|59711|
```

Comment:
Lets add in the ASC profiles:

- app-sample: 59700
- app-rules-engine: 59701
- app-push-to-core: 59702
- app-mqtt-export: 59703
- app-http-export: 59704
- app-functional-tests: 59705